src/game.js

```diff
@@ -0,0 +1,163 @@
+import Board from './factories/Board';
+
+const Game = (size, parent) => {
+  let turn = 0;
+  let boards;
+  let isGameOn = false;
+
+  const isGame = () => isGameOn;
+
+  const isOver = (board) => {
+    const ships = board.getShips();
+    return ships.every((s) => s.isSunk());
+  };
+
+  const enemyBoard = () => {
+    if (turn === 0) return 1;
+    return 0;
+  };
+
+  const getBoard = (id) => boards[id];
+
+  const getBoards = () => boards;
+
+  const switchTurn = () => {
+    if (!isGameOn) {
+      const winner = turn === 0 ? 'won' : 'lost';
+      alert(`GAME OVER, you ${winner}!`);
+      return;
+    }
```

Comment:
a restart button then appears upon which after click, page refreshes(to restart game)